spec/alephant/broker/cache/cached_object_spec.rb

```diff
@@ -0,0 +1,152 @@
+require "spec_helper"
+
+RSpec.describe Alephant::Broker::Cache::CachedObject do
+  subject { described_class.new(s3_obj) }
+
+  let(:last_modified) { Time.parse("Mon, 11 Apr 2016 10:39:57 GMT") }
+  let(:ttl)           { 15 }
+
+  let(:s3_obj) do
+    AWS::Core::Data.new(
```

Comment:
Yeah, I tend to try and use "real" objects in my tests as much as I can as i've been burned by over mocking/stubbing in the past.  But like you say, it's probably more idiomatic to use an instance_double.